### PR TITLE
Bug fixes & IPv6 support

### PR DESCRIPTION
**Pull request by JackSlateur**
Hi,

Please find a batch of commits I had to make

It mixes fixes (the code does not works for me as-is) and IPv6 support

Please tell me what you think, I can rebase it if you have issue with some parts or something

### CODE REVIEW

File: aioping/__init__.py [hunk 1]
```diff
@@ -211,8 +206,7 @@ def sendto_ready(packet, socket):
     )
     packet = header + data
 
-    callback = functools.partial(sendto_ready, packet=packet, socket=socket)
-    asyncio.get_event_loop().add_writer(my_socket, callback)
+    my_socket.sendto(packet, dest_addr)
```
Comment (anton-belousov) [inline aioping/__init__.py]: Why do we need a synchronous submission here?

**Review by anton-belousov — CHANGES_REQUESTED** [review]
Thank you for your changes! Everything is great, but I didn't get why do we need to use synchronous submission in when sending a request.

**Comment by JackSlateur** [comment]
Sorry for the delay and thanks for the review

I am not sure how it should be done, there is a commit that should do better

Tell me if it's still crap :)

File: aioping/__init__.py [hunk 2]
```diff
@@ -141,28 +147,41 @@ def checksum(buffer):
...
-            if type != 8 and packet_id == id_:
-                bytes_in_double = struct.calcsize("d")
-                time_sent = struct.unpack("d", rec_packet[28:28 + bytes_in_double])[0]
+                type, code, checksum, packet_id, sequence = struct.unpack(
+                    "bbHHh", icmp_header
+                )
 
-                return time_received - time_sent
+                if type != ICMP_ECHO_REPLY and type != ICMP6_ECHO_REPLY:
+                    next
```
Comment (bxwllzz) [inline aioping/__init__.py]: `next` is a build-in function in python. Do you mean `continue`?

**Review by bxwllzz — CHANGES_REQUESTED** [review]

**Comment by JackSlateur** [comment]
Humpf shame on me, thanks for the review

**Comment by anton-belousov** [comment]
@JackSlateur, @bxwllzz, thank you very much for contributing & checking this. Will update the library soon.

**Pull request merged**